snap/snapcraft.yaml

```diff
@@ -164,6 +164,11 @@ slots:
     content: aziotctl-executables
     source:
       read: [ $SNAP/bin ]
+  aziot-keys-library:
```

Comment:
I suggest calling it `aziot-keys-openssl-engine` so as to not confuse it with the other "aziot-keys library" that is the implementation used by keyd to interact with hardware.

Reply:
Updated